components/sidebar/sidebar_whats_new_modal/sidebar_whats_new_modal.scss

```diff
@@ -0,0 +1,31 @@
+@charset 'UTF-8';
+
+.GenericModal {
+    width: 692px;
+
+    
+    
+    .modal-content {
+        padding: 16px 15px;
+    }
+
+    .SidebarWhatsNewModal__helpText {
+        margin-top: 8px;
+    }
+
+    .SidebarWhatsNewModal__img {
+        margin-top: 40px;
+        height: 360px;
+        border: 1px solid var(--center-channel-color-16);
+        border-radius: 4px;
+        min-height: 360px;  // TODO: remove
```

Comment:
Is this something you meant to remove with this PR or later?

Reply:
I'm going to remove it once the GIF is actually in there, it's there to make sure the size of the modal is still the same.